### DISCUSSION TITLE
关键词标点符号问题

### DISCUSSION THREAD

**Discussion by CalvinWaWang** [sjtug/SJTUThesis#1009] [提问]
<img width="626" alt="截屏2024-01-24 下午4 22 59" src="https://github.com/sjtug/SJTUThesis/assets/33740878/993fd6b8-10d6-4dac-bd01-1618c006d96d">
<img width="258" alt="截屏2024-01-24 下午4 23 05" src="https://github.com/sjtug/SJTUThesis/assets/33740878/312e933d-9ae5-4e47-81e4-4dc1b82819fe">
如图，在中文关键词中用了英文的逗号，可最后显示的还是中文的逗号，请问如何解决？

**Comment by LogCreative** [comment]
配置并不对应于最终的输出结果，配置中必须使用英文逗号以区分不同的关键词。中文之间就应当使用中文逗号分隔，这就是正常的输出结果。
